website/docs/first-composite.mdx

```diff
@@ -121,7 +121,7 @@ composedb composite:models my-first-composite.json
 - Edit config file to add model ID to index
 - Restart Ceramic node
 
-## Interacting using GraphiQL
+## Interacting using GraphQL
```

Comment:
Graphiql is the tool used, so its correct, there is no typo mistake. Find picture attached as a proof
![Screenshot 2022-07-25 at 9 38 09 PM](https://user-images.githubusercontent.com/47674286/180826009-9226e258-fb2b-456d-9bae-47dc3ef0e2be.png)

Reply:
oh, my bad!